dgf/cronjobs.py

```diff
@@ -1,61 +0,0 @@
-import csv
```

Comment:
to lo que había en este archivo está separado en 3: `fetch_pdga_data`, `update_approved_discs` y `backup`

Reply:
en verdad esto mola porque así tb pueds llamar tu a los management command desde la consola haciendo simplemente `bash dgf.bash <nombre del comando>`